src/open_inwoner/plans/tests/test_views.py

```diff
@@ -571,22 +566,15 @@ def setUp(self) -> None:
         )
 
 
-@skip("skipped for now because of random CI failures, ref Taiga #963")
-class PlanActionStatusFirefoxSeleniumTests(
-    FirefoxSeleniumMixin,
-    _PlanActionStatusSeleniumMixin,
-    ActionStatusSeleniumBaseTests,
-    StaticLiveServerTestCase,
+class ChromePlanActionStatusPlaywrightTests(
+    _PlanActionStatusSeleniumMixin, ActionsPlaywrightTests
```

Comment:
We could rename the mixin above, since we won't have selenium in the project but we can do this when we add the test we are skipping for now.

Reply:
Oops, I was sloppy.